src/test/resources/pages/instructorFeedbackQuestionEditToTeamToTeam.html

```diff
@@ -978,38 +978,42 @@ <h4 class="modal-title">
                     <input checked="" class="visibilityCheckbox recipientCheckbox1" disabled="" name="receiverFollowerCheckbox" type="checkbox" value="RECEIVER">
                   </td>
                 </tr>
-                <tr>
-                  <td class="text-left">
-                    <div data-original-title="Control what team members of feedback giver can view" data-placement="top" data-toggle="tooltip" title="">
-                      Giver's Team Members
-                    </div>
-                  </td>
-                  <td>
-                    <input class="visibilityCheckbox answerCheckbox1" type="checkbox" value="OWN_TEAM_MEMBERS">
-                  </td>
-                  <td>
-                    <input class="visibilityCheckbox giverCheckbox1" type="checkbox" value="OWN_TEAM_MEMBERS">
-                  </td>
-                  <td>
-                    <input class="visibilityCheckbox recipientCheckbox1" type="checkbox" value="OWN_TEAM_MEMBERS">
-                  </td>
-                </tr>
-                <tr>
-                  <td class="text-left">
-                    <div data-original-title="Control what team members of feedback recipients can view" data-placement="top" data-toggle="tooltip" title="">
-                      Recipient's Team Members
-                    </div>
-                  </td>
-                  <td>
-                    <input class="visibilityCheckbox answerCheckbox1" type="checkbox" value="RECEIVER_TEAM_MEMBERS">
-                  </td>
-                  <td>
-                    <input class="visibilityCheckbox giverCheckbox1" type="checkbox" value="RECEIVER_TEAM_MEMBERS">
-                  </td>
-                  <td>
-                    <input class="visibilityCheckbox recipientCheckbox1" type="checkbox" value="RECEIVER_TEAM_MEMBERS">
-                  </td>
-                </tr>
+                <hide style="display: none;">
```

Comment:
I just checked the PR #5671. Turns out this is a change that should have been caught in that PR (meaning we do have a test for it after all, although not really strong).